recipes/scevan/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: scevan
+  version: 1.0.0  # Replace with the desired version
+
+source:
+  url: https://github.com/AntonioDeFalco/SCEVAN/archive/refs/tags/1.0.0.tar.gz  # Replace with the GitHub URL of the release
+  sha256: cc14b6b80019e06181bc69ab6f54bde9878b3b525645f925626bfed28df655fd  # Generate the checksum using 'sha256sum' command
+
+build:
+  number: 0
+  noarch: generic  # Specify that the package is noarch
+  run_exports: '{{ pin_compatible("SCEVAN", max_pin="x.x") }}'
+  rpaths:
+    - lib/R/lib
+    - lib
+  script: "$R -e \"install.packages(c('Rcpp', 'remotes', 'devtools'), lib='$PREFIX/lib/R/library', repos='https://cran.rstudio.com/')\""
+
+requirements:
+  build:
+    - r
+    - r-base
+    - r-remotes
+    - r-devtools
+  run:
+    - r
+    - r-base
+    - r-devtools
+    - r-paralleldist
+    - r-pheatmap
+    - r-forcats
+    - r-dplyr
+    - bioconductor-fgsea
+# - r-yaGST - manually with devtools
+    - r-cluster
+    - r-ggplot2
+    - r-Rtsne
+    - bioconductor-scran
+    - r-ape
+    - bioconductor-ggtree
+    - r-tidytree
+    - r-ggrepel
+    - r-optparse
+
+test:
+  commands:
+    - $R -e "library(devtools)"
+    - $R -e "install_github('miccec/yaGST')"
+    - $R -e "devtools::install_github('AntonioDeFalco/SCEVAN')"
```

Comment:
Hm!
I think is not correct!

1) This will install the latest code from the default branch. I think you need to specify the tag, e.g. `, ref = "1.0.0"`.
2) Why do you install it with devtools from Github at all ? IMO you should test the locally installed library instead

Reply:
will add the tag!

Reply:
This tag is fairly old though, around a year. I wonder if we can ask the maintainer to make a new release.

Reply:
updated to 1.0.1